spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/MessageBusConfiguration.java

```diff
@@ -54,6 +56,10 @@
 	 */
 	static ConfigurableApplicationContext createApplicationContext(final Properties properties) {
 		String transport = properties.getProperty("XD_TRANSPORT");
+		if (properties.getProperty(RABBIT_ACKMODE_PROPERTY) != null) {
+			properties.setProperty(RABBIT_ACKMODE_PROPERTY, "MANUAL");
+		}
```

Comment:
Should we even allow an override if `MANUAL` is required to make it work?

If not, we could either abort the deploy or just log a WARN.